layouts/components/announcement.html

```diff
@@ -6,10 +6,9 @@
  </svg>
 
      <p style="font-size:125%">
-     Registration and Abstract Submission Open for
      <br>
```

Comment:
Should we not still leave the sentence about registration being open, and just remove the Abstract Submission part?

```suggestion
     Registration Open for
     <br>
```